lib/transforms/caffeextension_legalizer.cc

```diff
@@ -178,7 +178,7 @@ static std::vector<Def> ConvertPool(const CAFFEExtensionInst* ext,
   int pad_h = FindAttributeValue(ext, "pad_h", pad);
   int pad_w = FindAttributeValue(ext, "pad_w", pad);
 
-  int round_mode = FindAttributeValue(ext, "round_mode", -1);
+  int round_mode = FindAttributeValue(ext, "round_mode", 0);
```

Comment:
use "1" as default value? So in line 206, we can setRoundMode(round_mode)

Reply:
the default  value of caffe is 0, which means ceil mode . so if we really get "0", "setRoundMode(round_mode)" is wrong

Reply:
ok